logstash-core/spec/logstash/acked_queue_concurrent_stress_spec.rb

```diff
@@ -0,0 +1,296 @@
+# encoding: utf-8
+require "logstash/util/wrapped_acked_queue"
+require "logstash/event"
+require "logstash/instrument/namespaced_metric"
+
+RSpec.configure do |config|
+  config.backtrace_exclusion_patterns << /rakelib\/test/
+  config.backtrace_exclusion_patterns << /vendor\/bundle/
+end
```

Comment:
What is this madness! Never seen it before!, But  I believe this setting will impact other example?